source/server-selection/server-selection.rst

```diff
@@ -579,14 +579,68 @@ Passing read preference to mongos and load balancers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If a server of type Mongos or LoadBalancer is selected for a read operation, the read
-preference is passed to the selected mongos through the use of the ``$readPreference``
-`Global Command Argument`_ according to the following rules:
+preference is passed to the selected mongos through the use of ``$readPreference``
+(as a `Global Command Argument` for OP_MSG or a query modifier for OP_QUERY) and, for
+OP_QUERY only, the ``SecondaryOk`` wire protocol flag, according to the following rules.
+
+For OP_MSG:
 
   - For mode 'primary', drivers MUST NOT set ``$readPreference``
 
   - For modes 'secondary', 'primaryPreferred', 'secondaryPreferred', and 'nearest', drivers
     MUST set ``$readPreference``
 
+For OP_QUERY:
```

Comment:
For readability, I'd suggest making this and "OP_MSG" above sub-headings. I think backticks would be the next level down from tildes. While this isn't really a problem for OP_MSG, which only has a list, the OP_QUERY section has quite a bit more content that a heading would help set apart.